main/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/bytebase/dbcost/client"
+	"github.com/bytebase/dbcost/client/aws"
+	"github.com/bytebase/dbcost/client/gcp"
+	"github.com/bytebase/dbcost/store"
+)
+
+var cloudProvider = make(map[store.CloudProvider]client.Client)
+
+func init() {
+	APIKeyGCP := os.Args[1]
+	cloudProvider[store.CloudProviderGCP] = gcp.NewClient(APIKeyGCP)
+	cloudProvider[store.CloudProviderAWS] = aws.NewClient()
+}
+
+const (
+	dirPath  = "./store/data"
+	fileName = "rds.json"
```

Comment:
Use var so that can generate the filePath. Also use path.Join instead of strings.Join().